### ISSUE TITLE
Create Web portal for registering chiefs

### ISSUE THREAD

**Issue by Obadha** [Obadha/baraza#13]
Chiefs will be registered by admins. We should have an online portal where this will happen. The idea is for chiefs not to register themselves, to prevent just anyone from registering themselves as chief!

The portal(website) will act as a window into the Baraza platform. Anyone who wants any information on Baraza, but are not necessarily users will come here. 

Cases might also be visible, but maybe identities of involved partied might be hidden. Food for thought.

**Tasks:** Create web registration portal for chiefs.
**Deliverables:** Registration portal.

**Comment by icarius06** [comment]
This could also double up as the landing page for the application.

**Comment by WinnieMakokha** [comment]
Can we tap into the sub-county commissioners to make this happen? Also, in twitter there's a model chief @ChiefKariuki is his twitter handle who has been using twitter to communicate to the residents of Lanet, Nakuru. He was awarded in the open data awards by ICT authority last year.

**Comment by Obadha** [comment]
@icarius06  yes let's do that. I might have to create another issue to describe what the landing page will look like.
